knowledge-base/wordsprocessing-measure-text-netframework

```diff
@@ -0,0 +1,171 @@
+---
+title: How to Measure Text in WordsProcessing .NET Framework
+description: Learn how to measure text using the WordsProcessing library in .NET Framework.
+type: how-to
+page_title: How to Measure Text in WordsProcessing .NET Framework
+slug: wordsprocessing-measure-text-netframework
+tags: word, docx, flow, measure, netframework, wordsprocessing 
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.1.305 | RadWordsProcessing |[Yoan Karamanov](https://www.telerik.com/blogs/author/yoan-karamanov)| 
+
+## Description
+
+This article shows how to measure text in [WordsProcessing]({%slug radwordsprocessing-overview%}) in the .NET Framework environment. 
+
+## Solution
+
+You can create a __Telerik.Windows.Documents.Fixed.Model.Editing.[Block]({%slug radpdfprocessing-editing-block%})__ instance with the same content and font properties as the [Run]({%slug radwordsprocessing-model-run%}) you want to insert. Then you can call the __Measure__ method of the block in order to obtain its measurements:
+
+```csharp
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Windows;
+using System.Windows.Media;
+using Telerik.Windows.Documents.Flow.FormatProviders.Docx;
+using Telerik.Windows.Documents.Flow.Model;
+using Telerik.Windows.Documents.Flow.Model.Editing;
+using Telerik.Windows.Documents.Spreadsheet.Model;
+
+namespace ConsoleNetFramework
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var flowDocument = new RadFlowDocument();
```

Comment:
Instead of **var**, you can use the class name, e.g. RadFlowDocument

Reply:
The See Also section is good, I would suggest adding links to the new KB articles in the respective docs article as well. Thus, it may be findable more easily for the end-users.